docs/changelog.md

```diff
@@ -1,5 +1,8 @@
 # Changelog
 
+### 2024-11-21
+
+- Catch RunnerError during runner creation.
```

Comment:
The changelog was meant for the users of the charm, not the developers.

I think line might be confusing for the users.

Reply:
Have removed the line